intake/gui/source/defined_plots.py

```diff
@@ -54,7 +54,7 @@ def setup(self):
         self.select = pn.widgets.Select(options=self.options, height=30,
                                         align='center', width=200)
         self.desc = pn.pane.Str()
-        self.pane = pn.pane.HoloViews(self._plot_object(self.selected))
+        self.pane = pn.Column(pn.panel(self._plot_object(self.selected)))
```

Comment:
Layouts will call pn.panel automatically if needed:

```suggestion
        self.pane = pn.Column(self._plot_object(self.selected))
```

Reply:
I'd probably also rename `pane` -> `panel` throughout.